v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/dbutils/dml/TypeHandler.java

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright (C) 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.templates.dbutils.dml;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.nio.ByteBuffer;
+import java.sql.Timestamp;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.stream.Collectors;
+
+class TypeHandler {
+    public static Boolean handleCassandraBoolType(String colName, JSONObject valuesJson) {
+        return valuesJson.getBoolean(colName);
+    }
+
+    public static Float handleCassandraFloatType(String colName, JSONObject valuesJson) {
+        return valuesJson.getBigDecimal(colName).floatValue();
+    }
+
+    public static Double handleCassandraDoubleType(String colName, JSONObject valuesJson) {
+        return valuesJson.getBigDecimal(colName).doubleValue();
+    }
+
+    public static ByteBuffer handleCassandraBlobType(String colName, JSONObject valuesJson) {
+        Object colValue = valuesJson.opt(colName);
+        if (colValue == null) {
+            return null;
+        }
+        return parseBlobType(colName, colValue);
+
+    }
+
+    private static ByteBuffer parseBlobType(String colName, Object colValue) {
+        byte[] byteArray;
+
+        if (colValue instanceof byte[]) {
+            byteArray = (byte[]) colValue;
+        } else if (colValue instanceof String) {
+            byteArray = java.util.Base64.getDecoder().decode((String) colValue);
+        } else {
+            throw new IllegalArgumentException("Unsupported type for column " + colName);
+        }
+
+        return ByteBuffer.wrap(byteArray);
+    }
+
+    public static Date handleCassandraDateType(String colName, JSONObject valuesJson) {
```

Comment:
What is the need of passing column name? We can just pass the value and get the new casted value in return? This will be for all the functions.

Reply:
as we are doing multiple parsing so it should be under the Handler Class